tests/testProcessFile.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+
```

Comment:
Per the recent RFC-215, please remove this shabang and make the file non-executable.